### PR TITLE
[ISSUE-2952] Change hamburger menu animation speed

### PR DESCRIPTION
**Pull request by arno-fukuda**
## Why:
Google Core Web Vitals rolled out a new [INP](https://developers.google.com/search/blog/2023/05/introducing-inp) score.
The mobile hamburger menu would occasionally get flagged for "need improvement".
Hypothetical change in hope that a lower duration for the `translare3d` transition will result in an overall better average score. 

Closes: 
https://github.com/shuding/nextra/issues/2952

## What's being changed (if available, include any code snippets, screenshots, or gifs):
For the hamburger menu sidebar, we lower the transition duration to 300ms. Prior it was not set and used a default that was anywhere between 400-500ms.

There could be value in exploring it further in the future to fully remove the transition. (As reference looking at Vercel Docs which are currently using Fumadocs)

## Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View
      deployment** link in this PR's timeline (this link will be available after
      opening the PR).

### CODE REVIEW

File: packages/nextra-theme-docs/src/components/sidebar.tsx [hunk 1]
```diff
@@ -406,8 +406,8 @@ export function Sidebar({
           showSidebar ? 'md:_w-64' : 'md:_w-20',
           asPopover ? 'md:_hidden' : 'md:_sticky md:_self-start',
           menu
-            ? 'max-md:[transform:translate3d(0,0,0)]'
-            : 'max-md:[transform:translate3d(0,-100%,0)]'
+            ? 'max-md:[transform:translate3d(0,0,0)] max-md:_duration-200'
+            : 'max-md:[transform:translate3d(0,-100%,0)] max-md:_duration-200'
```
Comment (dimaMachina) [inline packages/nextra-theme-docs/src/components/sidebar.tsx:410]: I feel this toooo fast I prefer to remove this `translate3d` in v4, can we keep the current behaviour for v3?

Comment (arno-fukuda) [reply packages/nextra-theme-docs/src/components/sidebar.tsx:410]: Fair enough!

Comment (dimaMachina) [reply packages/nextra-theme-docs/src/components/sidebar.tsx:410]: why closing? 🙂

Comment (arno-fukuda) [reply packages/nextra-theme-docs/src/components/sidebar.tsx:410]: I might have misunderstood.
So we want to keep things as they are for current v3, but revisit for v4?

Comment (dimaMachina) [reply packages/nextra-theme-docs/src/components/sidebar.tsx:410]: I am fine with reducing the duration of the hamburger menu and removing translate3d in v4. Does that sound good to you?

Comment (arno-fukuda) [reply packages/nextra-theme-docs/src/components/sidebar.tsx]: changed to 300ms, which seems to be a good middle.
Feel free to tweak the numbers if you feel it's still too fast.

File: packages/nextra-theme-docs/src/components/sidebar.tsx [hunk 2]
```diff
@@ -406,8 +406,8 @@ export function Sidebar({
           showSidebar ? 'md:_w-64' : 'md:_w-20',
           asPopover ? 'md:_hidden' : 'md:_sticky md:_self-start',
           menu
-            ? 'max-md:[transform:translate3d(0,0,0)]'
-            : 'max-md:[transform:translate3d(0,-100%,0)]'
+            ? 'max-md:[transform:translate3d(0,0,0)] max-md:_duration-300'
+            : 'max-md:[transform:translate3d(0,-100%,0)] max-md:_duration-300'
```
Comment (dimaMachina) [inline packages/nextra-theme-docs/src/components/sidebar.tsx:410]:
```suggestion
            ? 'max-md:[transform:translate3d(0,0,0)]'
            : 'max-md:[transform:translate3d(0,-100%,0)]'
```

**Review by dimaMachina — APPROVED** [review]

**Pull request merged**